src/lib/schematics/install/fonts/head-element.ts

```diff
@@ -33,7 +33,7 @@ export function appendElementToHead(host: Tree, project: WorkspaceProject, eleme
     throw `Could not find '<head>' element in HTML file: ${indexPath}`;
   }
 
-  const endTagOffset = headTag.sourceCodeLocation.endTag.startOffset;
+  const endTagOffset = headTag.sourceCodeLocation!.endTag.startOffset;
```

Comment:
Why is this never null? 

Similar for other one-off places (like `architect` below)

Reply:
This is never `null` because we explicitly enabled the `parse5` option to track the source code location. So we will always have this property set.

Regarding architect: We could do better here, but I just went with the assumption that every CLI project has an architect block configured. We also assert if the default `architect builders` are used in the `theming/index.ts` file.

Reply:
We should capture that explanation _somewhere_ in the code

Reply:
I will add a note for the `parse5` instances. There aren't too much of those.

For the architect. I'll probably create an utility function that does the checking for an architect target. That way we are strict-null check compliant and also make sure that projects that don't have a builder will gracefully exit with a meaningful error message.